### ISSUE TITLE
Rebrand DAO Dashboard - Toast notifications

### ISSUE THREAD

**Issue by peterjurco** [api3dao/api3-dao-dashboard#503]
Update styles of the toast messages based on the [rebranded designs](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=2218-3596&node-type=frame&m=dev)

**Comment by peterjurco** [comment]
Update: I have it implemented but there are many questions around toast functionality and design so I want to discuss it on the design call tomorrow

**Comment by peterjurco** [comment]
Currently blocked by https://github.com/api3dao/tasks/issues/1550

**Comment by T9991** [comment]
Designs: https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=10710-181899&t=GVTulrrWOfmXyT3S-1

The view transaction button should always have the external link icon

**Comment by Anboias** [comment]
Preview URL with testing buttons: https://bafybeiawvt4nyozvo2wpnciomcdkquchdhzcxmmvz6ieu3smkrmm6e25ke.ipfs.flk-ipfs.xyz/#/

<img width="1009" alt="Image" src="https://github.com/user-attachments/assets/1e13fd8b-e493-41fd-bb52-9719ceb54b40" />

**Comment by PrenaamD** [comment]
@Anboias is it intentional that the Pending transaction toast messages don't disappear automatically like the others

**Comment by Anboias** [comment]
@PrenaamD, yes. [So it seems](https://github.com/api3dao/api3-dao-dashboard/blob/c85bd9395c245995a95642e18d89a5dc26b6e36a/src/chain-data/helpers.ts#L37).

<img width="731" alt="Image" src="https://github.com/user-attachments/assets/aee845fb-9ea2-44a0-8e9d-6721606e9256" />
